pkgs/data/themes/nordic/default.nix

```diff
@@ -92,6 +91,8 @@ stdenvNoCC.mkDerivation rec {
 
   dontWrapQtApps = true;
 
+  dontCheckForBrokenSymlinks = true;
```

Comment:
Plsase add a comment link to the upstream issue like 

https://github.com/NixOS/nixpkgs/pull/380447/files#diff-dd269ad187d5fba444a8f7a2f231feb440bf32b50f777f6b6a84bc06a66392c3R34